packages/rest/src/coercion/coerce-parameter.ts

```diff
@@ -169,10 +169,37 @@ function coerceObject(input: string | object, spec: ParameterObject) {
   if (typeof data !== 'object' || Array.isArray(data))
     throw RestHttpErrors.invalidData(input, spec.name);
 
-  // TODO(bajtos) apply coercion based on properties defined by spec.schema
+  const schema = extractSchemaFromSpec(spec);
+  if (schema) {
+    // apply coercion based on properties defined by spec.schema
+    const ajv = new Ajv({coerceTypes: true});
```

Comment:
It's a bit concerning that we hard-code the instantiation of `Ajv` here. We typically want to inject the ajv options.

Reply:
@raymondfeng The ajv here is not used for validation, but for coercion purpose, I know this sounds a bit weird, but I don't find a better&simpler way to coerce based on schema.

See ajv doc in https://github.com/ajv-validator/ajv#coercing-data-types

If you search `options` in file [coerce parameter](https://github.com/strongloop/loopback-next/blob/4d22c725d2fe9e35663f0f50296602a7b6fa3d3f/packages/rest/src/coercion/coerce-parameter.ts), there is no options available for coercing object.

Reply:
We should add an optional `options` parameter for `coerceParameter`. Its main usage is in `parse.ts` which has access to the configuration.